src/utils.ts

```diff
@@ -52,7 +52,7 @@ export function toText(e: Error): string {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function getEnvResources(envFilePath: string): Promise<any> {
 	try {
-		if (fs.existsSync(envFilePath)) {
+		if (existsSync(envFilePath)) {
```

Comment:
I think fs.access() is a better choice

Reply:
fs.access throws exception in case of path not exists.